packages/@adobe/spectrum-css-temp/components/button/skin.css

```diff
@@ -410,6 +411,10 @@ governing permissions and limitations under the License.
 
     --spectrum-high-contrast-focus-ring-color: var(--spectrum-high-contrast-button-text);
   }
+
+  .spectrum-FieldButton {
+    --spectrum-high-contrast-focus-ring-color: var(--spectrum-high-contrast-highlight);
```

Comment:
this is directly overriding the same property set 2 lines up on this same selector, this could be hard to debug or understand in the future, instead, I propose we move the one above into its own. you can leave all the other properties common to all three together.
```
  .spectrum-Button,
  .spectrum-ActionButton {
    --spectrum-high-contrast-focus-ring-color: ButtonText;
  }
  
  .spectrum-FieldButton {
    --spectrum-high-contrast-focus-ring-color: Highlight;
  }
```